client/src/store/metadata/index.ts

```diff
@@ -142,6 +146,7 @@ export const useMetadataStore = create<MetadataStore>((set: SetState<MetadataSto
     getCurrentMetadata: (id: FileId): StateMetadata | undefined => {
         const { metadatas } = get();
         return metadatas.find(({ file }) => file.id === id);
+        console.log(metadatas);
```

Comment:
Is this console statement for debugging purposes or is it helpful to have for each run? If for debugging, be sure to remove it. If it should stay provide some context on the output by describing it. (e.g. 'metadata for file: ....')

Reply:
Confirmed: removed console statement.